README.rst

```diff
@@ -20,12 +20,19 @@ circuits and running them against quantum computers and simulators.
 Installation and Documentation
 ------------------------------
 
-Documentation is available at `cirq.readthedocs.io <https://cirq.readthedocs.io>`_:
+Cirq documentation is available at `cirq.readthedocs.io <https://cirq.readthedocs.io>`_.
+
+Documentation for the latest **unstable** version of cirq (tracks the repository's master branch; what you get if you ``pip install cirq-unstable``), is available at `cirq.readthedocs.io/latest <https://cirq.readthedocs.io/en/latest/>`_.
```

Comment:
extra credit: you could configure readthedocs to make the url `/unstable/` instead of `/latest/` for consistent terminology